src/manifestUtils.ts

```diff
@@ -68,6 +69,7 @@ export async function parseManifest(uri: Uri): Promise<Manifest | null> {
     }
 
     if (manifest === null) {
+        console.log(`Manifest at ${uri.fsPath} is not a valid Flatpak manifest`)
```

Comment:
Won't this be only be printed when the default is matched into?

Reply:
not necessarily, it could also happen if the json/yml parser fails for whatever reason

Reply:
I'm not sure how js error, but I think it would just throw error where the parser fails, not return, which is then catch by the caller